pkgs/development/compilers/eunomia-bpf/eunomia-cc/default.nix

```diff
@@ -0,0 +1,128 @@
+{ lib
+, stdenv
+, makeWrapper
+, fetchFromGitHub
+, rustPlatform
+, cmake
+, pkg-config
+, llvmPackages_16
+, cargo
+, rustc
+, elfutils
+, zlib
+, ...
+}:
+let
+  bpftool =
+    stdenv.mkDerivation {
+      pname = "bpftool";
+      version = "eunomia-edition-20230311";
+      src = fetchFromGitHub {
```

Comment:
So I see this is a fork of `libbpf` from https://github.com/libbpf/bpftool . Do you have an idea why enomia's team needs this fork?

Reply:
This fork made some changes to generate json format skeleton.

Reply:
> This fork made some changes to generate json format skeleton.

It'd have been nice if they had contributed those features upstream. Are you aware of such attempts? In case they are not interested in such contributions, I think the right thing to do would be to create an alternative `bpftools-enumia` attribute in `all-packages.nix`, with a `meta` attribute, mentioning this is a fork, and mention what features are not available upstream.

Reply:
> This is specialized for some functions and has finally been embedded into the ecc binary, is it really necessary to create a new package?
> 
> And as I know it may soon be rewritten and this fork may not maintained after that. [eunomia-bpf/eunomia-bpf#183](https://github.com/eunomia-bpf/eunomia-bpf/issues/183)

Thanks for the link. Considering it, it may indeed be the right thing to not add a global package in `all-packages.nix`. However, it'd be nice to put that link in a comment near where the package is defined, so other users of this package could track that issue.

Reply:
It'd also be nice to describe a bit in a comment near the link to https://github.com/eunomia-bpf/eunomia-bpf/issues/183 , why is there a fork in the first place.